tests/test_common.py

```diff
@@ -92,6 +93,28 @@ def test_is_writeable_array_numpy():
     assert not is_writeable_array(x)
 
 
+@pytest.mark.parametrize("library", all_libraries)
+def test_size(library):
+    xp = import_(library)
+    x = xp.asarray([1, 2, 3])
+    assert size(x) == 3
+
+
+@pytest.mark.parametrize("library", all_libraries)
```

Comment:
FIXME this doesn't actually test ndonnx's None case yet!

I need https://github.com/data-apis/array-api-compat/pull/227 to be merged before I can open a new PR specific to ndonnx.